lib/ssoe/configuration.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/base'
+
+module SSOe
+  class Configuration < Common::Client::Configuration::SOAP
+    def self.ssl_cert_path
+      IdentitySettings.ssoe_get_traits.client_cert_path
+    end
+
+    def self.ssl_key_path
+      IdentitySettings.ssoe_get_traits.client_key_path
+    end
+
+    def ssl_options
+      if ssl_cert && ssl_key
+        {
+          client_cert: ssl_cert,
+          client_key: ssl_key
```

Comment:
I don't see `ssl_cert` and `ssl_key` defined anywhere. We'll need to add methods that read the files and create the cert and key objects with `OpenSSL`. Also, should we even attempt to make a call if these don't exist?